easybuild/easyblocks/p/psi.py

```diff
@@ -184,9 +184,11 @@ def sanity_check_step(self):
     def make_module_extra(self):
         """Custom variables for PSI module."""
         txt = super(EB_PSI, self).make_module_extra()
-        psi4datadir = glob.glob(os.path.join(self.installdir, 'share', 'psi*'))
-        if len(psi4datadir) != 1:
-            raise EasyBuildError("Could not determine the PSI4 data dir, there are multiple possibilities: ",
-                                 psi4datadir)
-        txt += self.module_generator.set_environment('PSI4DATADIR', psi4datadir[0])
+        share_dir = os.path.join(self.installdir, 'share')
```

Comment:
the `glob` doesn't work if the directory is not there, e.g. with `--module-only --force` (which gets tested)